packages/contracts-bedrock/tasks/check-l2.ts

```diff
@@ -10,7 +10,7 @@ import { predeploys } from '../src'
 
 // expectedSemver is the semver version of the contracts
 // deployed at bedrock deployment
-const expectedSemver = '1.0.0'
+const expectedSemver = '1.1.0'
```

Comment:
This needs to be changed back to `1.0.0` and then the particular contract that was changed, `1.1.0` needs to be passed in as an override, so the `L2StandardBridge` section